Include/internal/pycore_interp.h

```diff
@@ -93,7 +93,6 @@ struct _Py_unicode_state {
 #  define PyTuple_MAXFREELIST 1
 #  define PyList_MAXFREELIST 0
 #  define PyDict_MAXFREELIST 0
-#  define PyFrame_MAXFREELIST 0
```

Comment:
Why are we removing the freelist? The commit doesn't explain anything for this

Reply:
The frameobjects are no longer a fixed size, so a free list wouldn't work.

We shouldn't be allocating many frameobjects anyway, they only get creating when an exception is raised, or explicitly requested.

Reply:
But this will happen when tracing/profiling, no? I suspect coverage runs for example will be slower due to this.

Reply:
Maybe, but freelists just don't work unless all objects are the same size.

Reply:
> 

I am skeptical of this, we are talking about the freelist of the frame object, which under coverage is going to be triggered a lot. I would like data to back this

Reply:
Which assertion? That the performance of coverage is already terrible, or that we need to fix it?

Reply:
This assertion:

> One freelist more or less isn't going to make a difference.

Reply:
I just want to land this having an idea of the impact, not to stop the PR for landing, just to be clear